db/migrate/20231129183721_add_title_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddTitleToPosts < ActiveRecord::Migration[7.1]
+  def change
+    add_column :posts, :title, :string
+  end
+end
```

Comment:
- [ ] [optional] Kindly consider removing this migration file, you have already create the title column in the `20231129101210_create_posts` file